language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/QNameRefCompletionUtil.java

```diff
@@ -126,7 +125,7 @@ public static List<Symbol> getTypesInModule(BallerinaCompletionContext context,
                 QNameRefCompletionUtil.getAlias(qNameRef));
         return module.map(symbol -> symbol.allSymbols().stream()
                 .filter(CommonUtil.typesFilter())
-                .collect(Collectors.toList()))
+                .toList())
```

Comment:
Same as above

Reply:
Fixed in [2778660](https://github.com/ballerina-platform/ballerina-lang/pull/42904/commits/277866060d74b08115089625bb7d582c095697a1)